bricks/template/__brick__/{{project_name.snakeCase()}}/lib/main.dart

```diff
@@ -6,11 +6,10 @@ import 'package:go_router/go_router.dart';
 import 'package:package_info_plus/package_info_plus.dart';
 import 'package:{{project_name.snakeCase()}}/di/di.dart';
 import 'package:{{project_name.snakeCase()}}/gen/assets.gen.dart';
-import 'package:{{project_name.snakeCase()}}/resources/app_colors.dart';
-import 'package:{{project_name.snakeCase()}}/usecases/user/get_users_use_case.dart';
-
-import 'home_view_model.dart';
-import 'home_view_state.dart';
+import 'package:{{project_name.snakeCase()}}/app/resources/app_colors.dart';
+import 'package:{{project_name.snakeCase()}}/app/screens/home/home_view_model.dart';
+import 'package:{{project_name.snakeCase()}}/app/screens/home/home_view_state.dart';
```

Comment:
I think it's time to separate the `HomeScreen` with the initial ViewModel and ViewState  setup to `app/screens/home`

Reply:
The `SecondScreen` could simply be put inside this main class.